bundle/Core/Search/SuggestionResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\Core\Search;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use Netgen\EzPlatformSearchExtra\API\Values\Content\Search\Suggestion;
+
+class SuggestionResolver
+{
+    /**
+     * Get suggested search term based on returned spell check suggestions.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Query $query
```

Comment:
No need for `@param` and `@return`.

Reply:
@emodric Fixed in https://github.com/netgen/site-bundle/pull/14/commits/3e8c09c8641d8f9c868449a70254273b20d7ad9e.